supabase_py/lib/Storage/RequestError.py

```diff
@@ -0,0 +1,11 @@
+class RequestError(Exception):
+    """ """
+
+    def __init__(self, status_code, error, message="An error accoured in the request "):
+        self.status_code = status_code
+        self.error = error
+        self.message = message
+        super().__init__(self.message)
+
+    def __str__(self):
+        return f'A request with status_code {self.status_code} gave an error: {self.error} with this message {self.message} '
```

Comment:
/nit standardized usage of quotes to "
```suggestion
        return f"A request with status_code {self.status_code} gave an error: {self.error} with this message {self.message} "
```